containerd-shim-v2/service.go

```diff
@@ -330,6 +367,9 @@ func (s *service) Cleanup(ctx context.Context) (_ *taskAPI.DeleteResponse, err e
 
 // Create a new sandbox or container with the underlying OCI runtime
 func (s *service) Create(ctx context.Context, r *taskAPI.CreateTaskRequest) (_ *taskAPI.CreateTaskResponse, err error) {
+	span, _ := trace(s.ctx, "Create")
```

Comment:
Don't you want to have a few `span.LogField()` added giving more information about what is being created?
(This same comment applies to practically all functions below)

Or do you plan to add that later?

Reply:
I think at least to start, the basic spans would be good -- it shows the trace of execution very clearly. if feasible, I think we should look at adding in follow on PR?

Reply:
@c3d @egernst we can consider this for a later PR